web/atlas.json

```diff
@@ -3824,7 +3823,6 @@
 {"id": "tx0pb0", "submitted_by": "Horsefur", "name": "Fursona of Riley Winters", "description": "The fursona of Riley Winters,a mountain lion.", "website": "https://twitter.com/Horsefur1", "subreddit": "/u/Horsefur", "center": [1500.5, 989.5], "path": [[1482.5, 982.5], [1482.5, 997.5], [1483.5, 998.5], [1496.5, 998.5], [1499.5, 993.5], [1524.5, 992.5], [1524.5, 988.5], [1515.5, 982.5], [1482.5, 983.5], [1482.5, 982.5]]},
 {"id": "tx0p7b", "submitted_by": "Vegaxys", "name": "Diamond shovel from Minecraft", "description": "A diamond shovel from the game Minecraft.", "website": "", "subreddit": "/r/Minecraft", "center": [1740.5, 348.5], "path": [[1746.5, 342.5], [1741.5, 342.5], [1731.5, 354.5], [1739.5, 354.5], [1749.5, 344.5]]},
 {"id": "tx0p6c", "submitted_by": "Virgilcore", "name": "Forgot to install CS:S", "description": "An ERROR model from games that run on the Source engine. The background is the MISSING TEXTURE from Source games. Most commonly associated with Garry's Mod.", "website": "", "subreddit": "/r/SourceEngine", "center": [1789.5, 229.5], "path": [[1773.5, 207.5], [1805.5, 207.5], [1805.5, 251.5], [1774.5, 251.5], [1774.5, 208.5], [1773.5, 208.5]]},
-{"id": "tx0p3o", "submitted_by": "AustinQ", "name": "Rick Roll QR Code", "description": "While it doesn't appear in this screencap, the Rick Roll QR Code was rebuilt here just a couple minutes before the end.", "website": "", "subreddit": "/r/placeQRcode", "center": [1887.5, 933.5], "path": [[1869.5, 917.5], [1905.5, 917.5], [1905.5, 948.5], [1869.5, 949.5], [1869.5, 917.5]]},

```

Comment:
I see discussion on r/placeQRcode about these coordinates. Even if your group did not communicate with them, it looks like they independently contributed. All parties should be credited, so keep that subreddit in the remaining non-duped entries.